### DISCUSSION TITLE
How to preserve the eraser's edits when cropping

### DISCUSSION THREAD

**Discussion by boomsi** [fabricjs/fabric.js#9382] [Q&A]
For this issue #8517 

```js
this.sendObjectToPlane(clipPath, undefined, newRect.calcTransformMatrix());
```

Now i get another question.
The element obtained after cropping does not include the eraser portion of the target element.

Before:
<img width="551" alt="image" src="https://github.com/fabricjs/fabric.js/assets/39713960/28a1141b-e38e-4d30-ab32-782a57b1ae62">

After:
<img width="544" alt="image" src="https://github.com/fabricjs/fabric.js/assets/39713960/0b7a7a9d-9dde-421d-8c48-12deff2456c5">

So how to preserve the eraser's edits when cropping?

**Comment by ShaMan123** [comment]
why open another thread and not continue there? It causes a mess
please continue there
